pallets/pool-system/src/tranches.rs

```diff
@@ -251,6 +254,7 @@ where
 				xcm: XcmMetadata {
 					fee_per_second: None,
 				},
+				transferability: Some(CrossChainTransferability::Connectors),
```

Comment:
What does it again mean, if a token in the registry has no `location` set. I mean, in our system?

Reply:
So, for a token to be transferrable through XCM it **must** have a `location`. If it doesn't have, and it supposed to be XCM-transferrable, it will not be actually transferrable; for Connectors-wrapped tokens, we discussed using the `location` as a type to encapsulate the corresponding EVM address of the token and it's evm chain id; so in that case having a `location` set, which would require having  `transferability: Some(CrossChainTransferability::Connectors)`, it would mean that the location served the purpose of holding that information that connectors would know how to unwrap.

Does that clarify or does it make it more confusing?